godot-ffi/src/binding/single_threaded.rs

```diff
@@ -10,13 +10,26 @@
 //! If used from different threads then there will be runtime errors in debug mode and UB in release mode.
 
 use std::cell::Cell;
+
+#[cfg(any(
+    not(target_family = "wasm"),
+    not(feature = "experimental-wasm-nothreads")
+))]
 use std::thread::ThreadId;
 
 use super::GodotBinding;
 use crate::ManualInitCell;
 
 pub(super) struct BindingStorage {
+    // No threading when linking against Godot with a nothreads Wasm build.
+    #[cfg(all(target_family = "wasm", feature = "experimental-wasm-nothreads"))]
+    main_thread_id: Cell<Option<()>>,
```

Comment:
do we even need the variable in case of no threads? Can't we remove it?

Reply:
Also, `Option<()>` is `bool` with extra steps :wink: 

Or is there existing code that just checks for presence without getting the inner value?

Reply:
Yeah, it was convenient to do this to avoid adding more cfgs where we check for initialization by checking for `Some` in other methods. We can replace the field with a simple bool one, though that'll lead to more conditional compilation. Maybe it doesn't matter that much though, since it'd be restricted to just this file.

Reply:
Would there be a lot to change?

If yes,`Option<()>` is _probably_ OK but then a comment should explain why not bool :slightly_smiling_face:

Reply:
I've changed it to bool. It wasn't that much tbh.